src/pages/nft/claim/index.vue

```diff
@@ -1098,11 +1098,15 @@ export default {
         this.primaryKey,
         1
       );
+      const query = { tab: 'collected' };
+      if (this.isAutoDeliver) {
+        query.refetch = true;
+      }
```

Comment:
Can we trigger a `fetchCollectedNFTClassesByAddress` here instead of in next page?

Reply:
How about trigger `fetchCollectedNFTClassesByAddress` right after claiming?
[🎨 Refetch classes list after claiming](https://github.com/likecoin/liker-land/pull/1631/commits/e73ee6eab252d3953d8f359e061167bf5dd5a3fa)